src/wrapped_radix_engine_toolkit/low_level/execution_summary/new_entities.rs

```diff
@@ -26,12 +26,12 @@ impl From<(RetNewEntities, NetworkID)> for NewEntities {
         Self::new(
             // We map from `IndexMap<GlobalAddress, IndexMap<String, Option<MetadataValue>>>`
             // into: `HashMap<ResourceAddress, NewlyCreatedResource>`,
-            // and "filter out" (ignore) any GlobalAddress that is not a ResourceAddress,
```

Comment:
let `ignore` primarily be used in `#[ignore]` of unit tests, allows for easy easy search of it.